ucs/adapters/inbound/fastapi_/routes.py

```diff
@@ -59,7 +59,12 @@
 }
 
 
-@router.get("/health", summary="health", status_code=status.HTTP_200_OK)
+@router.get(
+    "/health",
+    summary="health",
+    status_code=status.HTTP_200_OK,
+    tags=["UploadControllerService"],
+)
```

Comment:
You should be able to set common tags on the router like `router = APIRouter(tags=["UploadControllerService"])`, no need to declare them on every path separately.